bin/package-requires-update

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+var exec = require( 'child-process-promise' ).exec;
+var packageJson = require( '../' + process.argv[2] );
+var semver = require( 'semver' );
+
+if ( packageJson.private ) {
+	process.exitCode = 0;
+}
+
+if ( /[^A-Za-z0-9_\/:=-]/.test( packageJson.name ) ) {
+	console.log( packageJson.name, 'Package name cannot contain spaces and/or invalid characters' );
+	console.log( '\n' );
```

Comment:
You could roll the double line-feed into the console.log() on the line before, but it might be better to skip the padding.